sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientWrapper.scala

```diff
@@ -150,6 +152,21 @@ private[hive] class ClientWrapper(
     val original = Thread.currentThread().getContextClassLoader
     // Switch to the initClassLoader.
     Thread.currentThread().setContextClassLoader(initClassLoader)
+
+    val sparkConf = new SparkConf
```

Comment:
Instead of creating a new Spark Conf, can we use `SparkEnv.get.conf` to get the spark conf associated with the current spark context?